test/e2e/manual/docs/export_migrate.md

```diff
@@ -0,0 +1,91 @@
+## Export state and migrate to a new network
+
+The following steps perform a migration to a new local testnet, which inherits the chain state but
+starts from block height 0 again. This procedure is required for major upgrades that cannot be
+handled by a live upgrade.
+
+### Prerequisite
+
+Run `go run localnet.go -start -auto` to start testnet and auto config all nodes as validators.
+
+### Halt the chain at a predetermined block height
+
+1. Set the `halt-time` field in `../../../docker-volumes/nodeX/sgnd/config/app.toml` files to a future block height:
+
+2. Restart the nodes:
+
+```sh
+go run localnet.go -stopall
+go run localnet.go -upall
+```
+
+3. Wait for the nodes to halt.
+
+### Export chain state
+
+1. Stop the node processes:
+
+```sh
+go run localnet.go -stopall
+```
+
+2. Export the current state of the chain:
+
+```sh
+docker exec -it sgnnode0 sgnd export --config sgncli/config/sgn.toml --home sgncli --for-zero-height --height <last-commit-height> > sgnd/config/sgntest_genesis_export.json
```

Comment:
how could you `docker exec -it sgnnode0` after you stop the container in the previous step?

Reply:
Fixed.